windows/deploy_windows.ps1

```diff
@@ -256,6 +257,19 @@ function Build-App-Variants
 # Build Windows installer
 Function Build-Installer
 {
+    param(
+        [string] $BuildOption
+    )
+
+    if ($BuildOption -Eq "jackonwindows")
```

Comment:
Maybe good to add a comment here why we use a separate installer here.

Reply:
Will be done in next commit

Reply:
I'd like the "installer suffix" passed in.  If it's not empty, set the name to "installer_${InstallerSuffix}.nsi" otherwise "installer.nsi".  That's because the portaudio installer will also be separate and should have a different name.